qaframework-bdd-tests/src/test/resources/features/refapp-2.x/16-PhoneNumberValidator/phoneNumberValidator.feature

```diff
@@ -9,15 +9,15 @@ Feature:Phone Number validator
     When user searches for patient using patient identifier
     Then system loads patient dashboard page of last active visit
     And user clicks on show contact link
-    And user clicks on edit contact info
-    And user clicks on phone number edit
+    And user clicks on edit contact info link
+    And user clicks on phone number button
     And user clears phone number
     And user enters new phone number using script element
     And user clicks on confirm edit button
     Then system returns a message validator
     And user clears the phone number input area
     And user enters the correct phone number
     And user clicks again on confirm edit button
-    And user clicks on confirm patient
-    And user clicks on show contact
+    And user clicks on confirm patient button
+    And user clicks on show contact link
     Then user validate the correct number to be entered into the input area
```

Comment:
isn't it the system that validates the correct phone number?

Reply:
Sure thanks. Resolved.